mobile-app/src/services/message/message.interfaces.ts

```diff
@@ -0,0 +1,3 @@
+interface MessageService {
+	sendSMS(recipients: string[], body: string): Promise<void>;
```

Comment:
Ovde sam isao generickim putem posto se razlikuje sta vraca MessageService interface. Posto twilio vraca sms id(neku identifikaciju od poslatih poruka) pa sam mislio da je to mozda bitna stavka koju mozda hocemo negde u nekom trenutku persistovati.